sql/mysql-compatibility.md

```diff
@@ -78,38 +78,23 @@ TiDB supports most of the MySQL built-in functions, but not all. See [TiDB SQL G
 
 ### DDL
 
-TiDB implements the asynchronous schema changes algorithm in F1. The Data Manipulation Language (DML) operations cannot be blocked during DDL the execution. Currently, the supported DDL includes:
+In TiDB DDL does not block reads or writes to tables while in operation. However, some restrictions currently apply to DDL changes:
 
-+ Create Database
-+ Drop Database
-+ Create Table
-+ Drop Table
 + Add Index: Does not support creating multiple indexes at the same time.
```

Comment:
Don't support some operations about the generated column, like doing `add index` to a generated column.
Ref https://github.com/pingcap/docs/blob/master/v2.1/sql/generated-columns.md#limitations